doc/whats_new/v0.24.rst

```diff
@@ -91,6 +91,10 @@ Changelog
   when ``strategy='most_frequent'`` or ``strategy='constant'``.
   :pr:`17526` by :user:`Ayako YAGI <yagi-3>` and :user:`Juan Carlos Alfaro Jiménez <alfaro96>`.
 
+- |Feature| :class:`impute.SimpleImputer` now supports ``inverse_transform``
+  functionality to revert imputed data to original.
```

Comment:
You should mention that this is only when `add_indicator=True`.

Reply:
Updated